src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,51 +7,91 @@
 
 package frc.robot;
 
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
-import frc.robot.commands.ExampleCommand;
-import frc.robot.subsystems.ExampleSubsystem;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj.XboxController.Button;
+import frc.robot.commands.ToggleReaper;
+import frc.robot.commands.RunReaper;
+import frc.robot.commands.TankDrive;
+import frc.robot.commands.ToggleShifter;
+import frc.robot.misc2020.EnhancedJoystick;
+import frc.robot.subsystems.DriveBase;
+import frc.robot.subsystems.Reaper;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj.Solenoid;
 
 /**
- * This class is where the bulk of the robot should be declared.  Since Command-based is a
- * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
- * periodic methods (other than the scheduler calls).  Instead, the structure of the robot
- * (including subsystems, commands, and button mappings) should be declared here.
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a "declarative" paradigm, very little robot logic should
+ * actually be handled in the {@link Robot} periodic methods (other than the
+ * scheduler calls). Instead, the structure of the robot (including subsystems,
+ * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
-  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
+  EnhancedJoystick leftJoystick;
+  EnhancedJoystick rightJoystick;
+  XboxController manipulator;
 
+  public final DriveBase driveBase;
 
+  public final Reaper reaper;
   /**
-   * The container for the robot.  Contains subsystems, OI devices, and commands.
+   * The container for the robot. Contains subsystems, OI devices, and commands.
    */
   public RobotContainer() {
     // Configure the button bindings
     configureButtonBindings();
+
+    // Initialization things
+    leftJoystick = new EnhancedJoystick(Constants.LEFT_JOYSTICK_PORT);
+    rightJoystick = new EnhancedJoystick(Constants.RIGHT_JOYSTICK_PORT);
+    manipulator = new XboxController(Constants.MANIPULATOR_PORT);
+
+    final VictorSP reaperMotor;
```

Comment:
I misled you actually. This motor doesn't need to be inverted. It's the thumbsticks on the xbox controller that behave weirdly. You can just directly pass the constructor a `new VictorSP(Constants.REAPER_MOTOR_PORT)` object.

Reply:
For the Thumbsticks Y value and the big joystick's Y value, down is positive and up is negative. Interesting.

Reply:
Disregard the first comment. This part of your code does not need to change. This is quite a day.